### PR TITLE
docs: Mention Sass variables and mixins in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mayank99**
- Added mention of existing Sass variables
- Added link to themed mixin and improved example

### CODE REVIEW

**Review by veekeys — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ We provide a script that can automatically create all the necessary files for yo
 
 - Add all your component styles in `src/[component-name]/[component-name].scss` file.
   - Break variables and mixins into separate files where possible.
+  - Use existing Sass variables for spacing (margin, padding, sizing, etc).
+  - Use [`themed`](https://github.com/iTwin/iTwinUI/blob/main/src/style/theme.scss#L430) mixin wherever color is involved.
```
Comment (bentleyvk) [inline CONTRIBUTING.md:70]: Maybe add an example how to use it.

Comment (mayank99) [reply CONTRIBUTING.md:70]: themed mixin has example comment above it

Comment (veekeys) [reply CONTRIBUTING.md:70]: There is one if you click the link.

Comment (mayank99) [reply CONTRIBUTING.md:70]: Improved the example

**Pull request merged**